src/services/log.ts

```diff
@@ -432,11 +457,70 @@ export function makeCopyBtn(v: any, copyBag: Array<any>): string {
   return `<span class="icon copy-btn tooltip is-tooltip-right" data-tooltip="Copy" data-copy="${copyBag.length - 1}"><i class="mdi mdi-content-copy"></i></span>`
 }
 
-export const transformers: { [key: string]: (any) => Transform } = {
+interface JavaExceptionMatch {
+  // If a fqcn has this prefix, it will turn into a link.
+  fqcnMatch: string
+  // The link generated for a matched fqcn
+  href: string
+}
+
+interface JavaExceptionConfig {
+  matches: JavaExceptionMatch[]
+}
+
+function fqcnMatches(fqcn: string, config: JavaExceptionConfig): JavaExceptionMatch | undefined {
```

Comment:
inline this function please, easier to read that way

Reply:
https://github.com/cashapp/logquacious/pull/23